EIPS/eip-4959.md

```diff
@@ -0,0 +1,325 @@
+
+#### Sidechain
+
+##### Ingredients
+
+ * **`50 ml`** cognac
+ * **`20 ml`** orange liqueur
+ * **`20 ml`** lemon juice
+
+**Garnish** cocktail glass rimmed with sugar
+
+##### Preparation
+
+1. In a cocktail shaker, combine all the ingredients.
+1. Shake.
+1. Strain into a cocktail glass rimmed with sugar.
+
+#### `SLOAD` Fashioned
+
+##### Ingredients
+
+ * **`50 ml`** bourbon
+ * **`10 ml`** maraschino liqueur
+ * **`20 ml`** ginger ale
+ * **`5 ml`** Angostura bitters
+
+**Garnish**
+
+ * **`1 slice`** orange
+ * **`1`** cocktail cherry
+
+##### Preparation
+
+1. Combine maraschino liqueur, ginger ale, and Angostura bitters in an old-fashioned glass.
+1. Fill the glass with ice.
+1. Add bourbon.
+1. Garnish with an orange slice and cocktail cherry.
+
+#### Ramos DeFizz
+
+##### Ingredients
+
+ * **`45 ml`** gin
+ * **`2 slices`** orange
+ * **`45 ml`** lime sour syrup
+ * **`15 ml`** lemon juice
+ * **`60 ml`** cream
+ * **`1`** egg white
+ * **`0.1 ml`** vanilla extract
+ * **`140 ml`** soda water
+
+ **Garnish** **`1 bean`** vanilla
+
+##### Preparation
+
+1. In a cocktail shaker, thoroughly muddle the orange slices and lime sour syrup.
+1. Add gin, lemon juice, cream, egg white, and vanilla extract.
+1. Shake, without ice, for two minutes.
+1. Remove orange slices.
+1. Add three 2.5 cm ice cubes, shake hard until ice is melted.
+1. Strain into a highball glass.
+1. Fill glass with soda.
+1. Garnish with vanilla bean.
+
+#### Rusty Node
+
+##### Ingredients
+
+* **`40 ml`** scotch whisky
+* **`20 ml`** drambuie
+* **`40 ml`** iced coffee
+
+**Garnish** None
+
+##### Preparation
+
+1. Combine all ingredients in an old-fashioned glass over ice.
+1. Stir gently.
+
+## Rationale
+
+The Blocktail primarily features Earl Grey tea to honor the location of the first Ethereum Devcon conference (London, GB.)
+
+All other drinks in this EIP are humorous takes on preexisting beverages, many inspired by International Bartenders Association classics.
+
+## Backwards Compatibility
+
+The drinks specified in this EIP use standard ingredients available in most venues.
+
+In the interest of international applicability, this EIP uses metric units over imperial measurements, which is different from (but compatible with) standard North American cocktail recipes.
+
+## Security Considerations
+
+There are negative health effects related to consuming alcohol, and standardizing a drink menu implies a certain level of support for doing so. By finalizing this EIP, the community confirms this support. It should be noted that this EIP _only_ exists to standardize a drink menu for those who would already be consuming alcohol, and does not aim to increase alcohol consumption at events.
+
+In addition to the above longer term health effects, alcohol has an immediate effect causing impairment. No development or research on the Ethereum protocol or related standards should be performed while under the influence of an excess of these beverages.
```

Comment:
Should these effects be included in a test cases section?

Reply:
Hm, I think the tests in Test Cases verify that the EIP has been implemented correctly, so for this proposal they'd be taste tests?

Reply:
Also, cases seem to be a bit too much to test. How about we only need to test 50mL?